synapse/cortex.py

```diff
@@ -208,9 +208,10 @@ class Cortex(s_cell.Cell):
 
     cellapi = CoreApi
 
-    def __init__(self, dirn):
+    def __init__(self, dirn, conf={}):
```

Comment:
@vEpiphyte - do we want to do something else here? Looks like configable went away

Reply:
Does everything need to be specified in the dmon.yml now?

Reply:
yes - everything for a cell comes in via a file. This is why the test helper `getTestCell` exists,

Reply:
allowing a conf coming in from an api config gets us back to a multimaster problem

Reply:
fixed, ty